tests/boot/boot_from_pxe.pm

```diff
@@ -64,7 +64,7 @@ sub run {
         my $repo        = $openqa_url . "/assets/repo/${image_name}";
         my $sut_machine = get_var('SUT_IP', 'nosutip');
         my $key_used    = '';
-        if (is_remote_backend && check_var('ARCH', 'aarch64') && ($sut_machine =~ /arch\.suse\.de/img)) {
+        if (is_remote_backend && check_var('ARCH', 'aarch64') && ($sut_machine =~ /(arch\.suse\.de|qa2\.suse\.asia)$/img)) {
```

Comment:
Please import `is_aarch64` and use it :) 

Also you could make the sentence more legible by using a `qr//`
```suggestion
        my $allowed_sut_re = qr/(arch\.suse\.de|qa2\.suse\.asia)/im;
        if (is_remote_backend && is_aarch64 && ($sut_machine =~ $allowed_sut_re)) {
```

Reply:
Done